skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +355,136 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
```

Comment:
Ping on these tests:

- `table` and `grouping` that have same samples but different order should give same results as a test where they're ordered the same. Basically testing that `table` and `grouping` indexes get aligned (one of the really cool features of pandas, and an important unit test to include!)

- specifying different tau, alpha, and theta values that are valid